### PR TITLE
Update Flyte Console Contribution Guide

### PR DESCRIPTION
**Pull request by samhita-alla**
## _Read then delete this section_

Updated README and added contribution guide.

_- Make sure to use a concise title for the pull-request._

_- Use #patch, #minor or #major in the pull-request title to bump the corresponding version. Otherwise, the patch version
will be bumped. [More details](https://github.com/marketplace/actions/github-tag-bump)_

# TL;DR
_Please replace this text with a description of what this PR accomplishes._

## Type
 - [ ] Bug Fix
 - [ ] Feature
 - [ ] Plugin

## Are all requirements met?

 - [ ] Code completed
 - [ ] Smoke tested
 - [ ] Unit tests added
 - [ ] Code documentation added
 - [ ] Any pending items have an associated Issue

## Complete description
 _How did you fix the bug, make the feature etc. Link to any design docs etc_

## Tracking Issue
_Remove the '*fixes*' keyword if there will be multiple PRs to fix the linked issue_

fixes https://github.com/flyteorg/flyte/issues/<number>

## Follow-up issue
_NA_
OR
_https://github.com/flyteorg/flyte/issues/<number>_

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
+# Flyte Console Contribution Guide
```
Comment (samhita-alla) [inline CONTRIBUTING.md:1]: I believe this file needs to be further simplified for first-time contributors. If possible, we need to have a step-by-step guide explaining the contribution process. Also, please check if the current content fits into the contribution guide. :)

File: README.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+  </a>
+  <a href="https://codecov.io/gh/lyft/flyteconsole">
+      <img src="https://img.shields.io/codecov/c/github/lyft/flyteconsole.svg" alt="Code Coverage" />
+  </a>
+</p>
+
+## 📦 Install Dependencies
+Running flyteconsole locally requires [NodeJS](https://nodejs.org) and
+[yarn](https://yarnpkg.com). Once these are installed, all of the dependencies
+can be installed by running `yarn` in the project directory.
```
Comment (anrusina) [inline README.md:34]: by running **`yarn install`**

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+First off, thank you for thinking about contributing! 
+Below you’ll find instructions that will hopefully guide you through how to contribute to, fix, and improve Flyte Console.
+
+## Storybook
+
+This project has support for [Storybook](https://storybook.js.org/).
+Component stories live next to the components they test, in a `__stories__`
+directory, with the filename pattern `{Component}.stories.tsx`.
+
+You can run storybook with `yarn run storybook`, and view the stories at http://localhost:9001.
```
Comment (anrusina) [inline CONTRIBUTING.md:12]: `yarn storybook` would also be sufficient

**Review by anrusina — APPROVED** [review]
Merged with master and updated appropriately

**Comment by samhita-alla** [comment]
> Merged with master and updated appropriately

Thank you, @anrusina! Sorry I missed your suggestions.

**Comment by anrusina** [comment]
No worries 😉 
It was easy to miss my comment keeping in mind that PR was open quite a time ago 😃 
Thanks a lot for doing all these changes, they look pretty nice

**Pull request merged**